bin/cic

```diff
@@ -3,6 +3,8 @@
 CURRENT_PATH="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 # shellcheck source=bin/support/sh/functions/cic_docker.sh
 source "${CURRENT_PATH}/support/sh/functions/cic_docker.sh"
+# shellcheck source=bin/support/sh/functions/cic.sh
```

Comment:
Nice - I like that you've made the effort to tell shellcheck where to find the files we're sourcing in.